eslintrc.json

```diff
@@ -62,7 +62,180 @@
     "no-path-concat": 0,
     "no-throw-literal": 0,
     "no-console": 1,
-    "import/no-extraneous-dependencies": 1
+
+
+    "accessor-pairs": "error",
```

Comment:
I have no idea what most of these are. I copied the rules from https://github.com/standard/eslint-config-standard/blob/master/eslintrc.json and deleted the duplicates.